src/main/java/uk/gov/digital/ho/hocs/info/security/KeycloakService.java

```diff
@@ -54,23 +55,29 @@ public KeycloakService(
     public CreateUserResponse createUser(CreateUserDto createUserDto) {
 
         UsersResource usersResource = keycloakClient.realm(hocsRealmName).users();
-
-        if (usersResource.search(createUserDto.getEmail()).size() > 0) {
-            log.warn("User {} creation failed - email already exists", createUserDto.getEmail(),
-                    value(EVENT, KEYCLOAK_FAILURE));
-            throw new ApplicationExceptions.UserAlreadyExistsException();
+        List<UserRepresentation> existingUserReps = usersResource.search(createUserDto.getEmail());
+
+        if (!existingUserReps.isEmpty()) {
+            log.info("Event: {}. Status: {}. User with email already exists as user {}",
+                LogEvent.CREATE_USER_FAILED,
+                HttpStatus.SC_CONFLICT,
+                existingUserReps.get(0).getId()
+            );
+            throw new ApplicationExceptions.UserAlreadyExistsException("User with provided email address already exists", LogEvent.CREATE_USER_FAILED);
         }
 
         UserRepresentation userRepresentation = mapToUserRepresentation(createUserDto);
-
         Response response = usersResource.create(userRepresentation);
+
         if (response.getStatus() != HttpStatus.SC_CREATED) {
             String errorMessage = extractCreateUserErrorMessage(response.getEntity());
-            log.error("Failed to create user {}, status {}", createUserDto.getEmail(), response.getStatus(), value(EVENT, KEYCLOAK_FAILURE));
+            log.error("Failed to create user, status {}", response.getStatus(), value(EVENT, KEYCLOAK_FAILURE));
             throw new KeycloakException(errorMessage, response.getStatus());
         }
 
         String userId = CreatedResponseUtil.getCreatedId(response);
+        log.info("Event: {}. Created new user {}", LogEvent.CREATE_USER_SUCCESS, userId);
```

Comment:
Can this be altered - IntelliJ will complain but allows it to be picked up by Kibana.

```suggestion
        log.info("Created new user {}", userId, value(EVENT, LogEvent.CREATE_USER_SUCCESS));
```